{"links": {"self": {"href": "data/repositories/levitsky/pyteomics/pullrequests/12/comments/43608520.json"}, "html": {"href": "#!/levitsky/pyteomics/pull-requests/12/_/diff#comment-43608520"}}, "deleted": false, "pullrequest": {"type": "pullrequest", "id": 12, "links": {"self": {"href": "data/repositories/levitsky/pyteomics/pullrequests/12.json"}, "html": {"href": "#!/levitsky/pyteomics/pull-requests/12"}}, "title": "added preindexing scheme and tests"}, "content": {"raw": "Thank you!\n\nBefore I accept this, I would like to find out why Python 3.6 complains about an unclosed file at `work_path` after running `test_prebuild_index`, despite there being a `close()` call after every use of `inst`. I wouldn't even think these were necessary after a `with` statement.\n\nI'm not sure what's going on, especially since I don't see the warning when I comment out all methods but `test_prebuild_index`, but I do see it when I also uncomment another method (doesn't seem to matter which).\n\n", "markup": "markdown", "html": "<p>Thank you!</p>\n<p>Before I accept this, I would like to find out why Python 3.6 complains about an unclosed file at <code>work_path</code> after running <code>test_prebuild_index</code>, despite there being a <code>close()</code> call after every use of <code>inst</code>. I wouldn't even think these were necessary after a <code>with</code> statement.</p>\n<p>I'm not sure what's going on, especially since I don't see the warning when I comment out all methods but <code>test_prebuild_index</code>, but I do see it when I also uncomment another method (doesn't seem to matter which).</p>", "type": "rendered"}, "created_on": "2017-08-25T14:44:45.733004+00:00", "user": {"display_name": "Lev Levitsky", "uuid": "{eb44325f-4ee0-4e0b-a27c-f2ea23122a56}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Beb44325f-4ee0-4e0b-a27c-f2ea23122a56%7D"}, "html": {"href": "https://bitbucket.org/%7Beb44325f-4ee0-4e0b-a27c-f2ea23122a56%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/a2593c44c42429c503d2e5e9e307e241d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsLL-6.png"}}, "nickname": "levitsky", "type": "user", "account_id": "557058:986c547b-c50a-40b3-948a-29b4a93b7b30"}, "updated_on": "2017-08-25T14:44:45.734884+00:00", "type": "pullrequest_comment", "id": 43608520}